admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -29,8 +29,13 @@
 			$(".ajax_table_link").click(function () {
 				var link = $(this);
 				$.post($(this).attr('href'), function (data) {
-					if (data.success == 1) {
-						showSuccessMessage(data.text);
+				  // If response comes from symfony controller
+          // then data has "status" and "message" properties
+          // otherwise if response comes from legacy controller
+          // then data has "success" and "text" properties.
+
+					if (data.success == 1 || data.status === true) {
+						showSuccessMessage(data.text || data.message);
```

Comment:
`data.text || data.message` JS trick, eh 😄 ?

Reply:
Use `data.text` if not null / undefined use `data.message`, it works on Bash and many other languages :)

Reply:
Can you change all tabs by spaces in a new commit?

Reply:
> Can you change all tabs by spaces in a new commit?

You mean indent?